pkg/api/mlflow/service/artifact/storage/helpers.go

```diff
@@ -0,0 +1,11 @@
+package storage
+
+import "strings"
+
+// PrepareS3Prefix transforms input path to correct S3 prefix.
+// `runArtifactURI` represents path in next format -> s3://fasttrackml/2/30357ed2eaac4f2cacdbcd0e06e9e48a/artifacts.
+// to correctly pass it to S3, we have to remove `s3://fasttrackml/` prefix,
+// so final result has to be -> 2/30357ed2eaac4f2cacdbcd0e06e9e48a/artifacts.
+func PrepareS3Prefix(basePath, runArtifactURI string) string {
```

Comment:
maybe this "ExtractsS3Path" instead of "PreparesS3Prefix"

Reply:
yeeep. this name is much better.